### PR TITLE
Issue/508/rename window api and fix api imports

### PR DESCRIPTION
**Pull request by Bedpaw**

### CODE REVIEW

**Comment by Bedpaw** [comment]
- Add initDevTools function, where all dev mode changes are initilized.
- Add required api imports in some files

File: src/components/organisms/map.vue [hunk 1]
```diff
@@ -15,6 +15,7 @@ import { mapMutations } from 'vuex';
 import { toLonLat } from 'ol/proj';
 import OPopupMap from 'organisms/popup/map';
 import Cookies from 'js-cookie';
+import { api } from '../../api';
```
Comment (dbetka) [inline src/components/organisms/map.vue:18]: Webpack has define alias for api. You can use it like below:
```
import { api } from 'api';
```

File: src/config/dev-tools.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import { api } from '../api';
+import { autoLogin } from '../utils/dev-mode/auto-login';
```
Comment (dbetka) [inline src/config/dev-tools.js:2]: same like above:
```
import { autoLogin } from 'utils/dev-mode/auto-login';
```

File: src/config/index.js [hunk 3]
```diff
@@ -1,7 +1,11 @@
 import { styleManager } from 'utils/style-manager';
 import uuidInit from 'utils/uuid';
+import { initializeDevTools } from './dev-tools';
 
 export function initApp () {
+  if (PRODUCTION === false) {
```
Comment (dbetka) [inline src/config/index.js:6]: I have some doubts about putting it into config dir. Have you any rule what decide what should be in config path and what not?

**Review by dbetka — CHANGES_REQUESTED** [review]
Idea with devTools is great! Good job!

File: src/config/index.js (hunk 3 above)
Comment (Bedpaw) [reply src/config/index.js:6]: It is in building phase. Imo in config we should keep configurations and initialization of external resources and our buisnes logic config. In future we can split in further, but now i dont see better place. If you have any idea, please share it here.

Comment (dbetka) [reply src/config/index.js]: > It is in building phase. Imo in config we should keep configurations and initialization of external resources and our buisnes logic config. In future we can split in further, but now i dont see better place. If you have any idea, please share it here.

I thing it's ok for now

**Comment by dbetka** [comment]
@Bedpaw Remember about README update. 🙂

**Pull request merged**